CONTRIBUTING.md

```diff
@@ -5,92 +5,70 @@
 
 ## I don't want to read this whole thing, I just have a question
 
 > **Note:** Please don't file an issue to ask a question. You'll get faster results by reaching out to the community on our [Discord channel](https://cloudquery.io/discord)
 
-
 ## What To Know Before Getting Started
 
 ### CloudQuery Architecture
 
-CloudQuery has a pluggable architecture and is using the go-plugin to load, run, and communicate between providers via gRPC. To develop a new provider for CloudQuery, you don’t need to understand the inner workings of go-plugin as those are abstracted away via the [provider-sdk](#cq-provider-sdk-repo).
-
-![architecture overview](https://www.cloudquery.io/images/cloudquery-architecture.png)
+CloudQuery has a pluggable architecture and is using [gRPC](https://grpc.io/) to communicate between source plugins, CLI and destination plugins. To develop a new plugin for CloudQuery, you don’t need to understand the inner workings of gRPC as those are abstracted away via the [plugin-sdk](#cloudquery-plugin-sdk-repository).
 
 ### Breakdown of Responsibilities and Repositories
 
-#### CQ CLI [Directory](https://github.com/cloudquery/cloudquery/tree/main/cli):
-- Main entry point and CLI for the user
-- Reading CloudQuery configuration
-- Downloading, verifying, and running plugins
-
-#### CQ Provider SDK [Repository](https://github.com/cloudquery/cq-provider-sdk):
-- Interacting with CQ CLI for initialization and configuration
-- Helper functions for defining table schemas
-- Methods for testing resources
-- Framework for running and building a plugin locally
+#### CloudQuery CLI [Directory](./cli)
 
-#### CQ Plugins
-- [Officially-supported Plugins](https://github.com/cloudquery/cloudquery/tree/main/plugins) and [Community plugins](https://github.com/search?p=1&q=cq-provider-&type=Repositories)
-- Previously known as Providers
-- Executed by CQ CLI via gRPC 
-- Interaction with remote data sources:
-    - Initialization of clients
-    - Authentication
-    - Fetching of configuration information
-- More information about developing your own plugin can be found [here](https://docs.cloudquery.io/docs/developers/developing-new-provider)
+* Main entry point and CLI for the user
+* Reading CloudQuery configuration
+* Downloading, verifying, and running plugins
 
+#### CloudQuery Plugin SDK [Repository](https://github.com/cloudquery/plugin-sdk)
 
-## How Can I Contribute?
+* Interacting with CloudQuery CLI for initialization and configuration
+* Helper functions for defining table schemas
+* Methods for testing the resource
+* Framework for running and building a plugin locally
 
-### Reporting Bugs and Suggesting Enhancements
+#### CloudQuery Plugins
 
-See [issue_reporting.md](issue_reporting.md) for more details on how to do this.
```

Comment:
Probably more familiar to keep the information on how to submit issues in the issues templates